### DISCUSSION TITLE
#Comofaz?

### DISCUSSION THREAD

**Discussion by mikaelmonteirodev** [dcTeam7/Course#108] [Q&A] [answered]
My friends! Uma dúvida:

Quando encontrarmos algo que pode ser corrigido nos projetos, qual seria a boa prática a ser utilizada?

1. Abrir uma _**issue**_ informando o problema e dando uma sugestão de correção?
2. Informar aos gerentes de projeto sobre o problema e a sugestão de correção pelo _**Wtsapp/Discord**_ ?
3. Clonar o repositório e alterar diretamente a _**branch main**_ 👀 ?
4. Outra opção: Diz aí...

No aguardo! 👍 
@AbraaoAlves @alanafilgueiras @fabrjcio @joellacerda @KevinDik @Deborahhm @gabrielapcustodio @rafaelspanholi @AntonioCaiodev @junioralencardev @Paulo23ox @AlexandreDEVNunes @andremoreiradev @emidiocom @JoelRamosdev

**Comment by kevinDsousa** [comment, marked as answer]
oii @mikaelmonteirodev, acredito que caso seja visto um bug, para que possamos documentar,  seria interessante criar uma issue no backlog com flag de bug onde voce descreve o problema.

**Reply by mikaelmonteirodev** [reply]
Show!

**Comment by AbraaoAlves** [comment]
[Issues](https://github.com/features/issues) são uma forma facil de gerenciar ideias, bugs e tarefas a fazer em um mesmo lugar. 

Algo interessante que devemos fazer é deixar um template para bugs e sugestão de features para que todos tenham uma mesmo formato.
Isso pode ser feito com ajuda dos issues_templates, como feito nesse [projeto do typescirpt](https://github.com/microsoft/TypeScript/tree/main/.github/ISSUE_TEMPLATE)
